include/netcdf.h

```diff
@@ -1948,7 +1948,7 @@ ncrecget(int ncid, long recnum, void **datap);
 EXTERNL int
 ncrecput(int ncid, long recnum, void *const *datap);
 
-  //EXTERNL int nc_finalize();
+/* EXTERNL int nc_finalize(); */
```

Comment:
I would suggest you just delete this line. It's extremely historical, I believe. There has never been an nc_finalize in a released netCDF version, as far as I am aware.